content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -444,6 +444,39 @@ securityContext:
 To assign SELinux labels, the SELinux security module must be loaded on the host operating system.
 {{< /note >}}
 
+### Speeding up SELinux volume relabeling
```

Comment:
```suggestion
### Efficient SELinux volume relabeling
```
“speeding up” might not make as much sense once the feature is stable. We like to write [timeless](https://developers.google.com/style/timeless-documentation) documentation if we can.

Reply:
Great suggestion! Applied